File: app/src/main/java/com/friendlypos/login/activity/ConfiguracionActivity.java
package com.friendlypos.login.activity;

import android.content.Intent;
import android.os.Bundle;
import android.support.design.widget.CoordinatorLayout;
import android.support.design.widget.FloatingActionButton;
import android.support.design.widget.Snackbar;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import android.view.View;
import android.widget.TextView;
import android.widget.Toast;

import com.friendlypos.R;
import com.friendlypos.application.util.Functions;
import com.friendlypos.login.util.Properties;

public class ConfiguracionActivity extends AppCompatActivity {
    TextView webServiceUrl;
    FloatingActionButton save;
    Properties properties;
    String nombreURL;
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_configuracion);
        properties = new Properties(this);
        webServiceUrl = (TextView) findViewById(R.id.txtwebservice);

        nombreURL = properties.getUrlWebsrv();

        webServiceUrl.setText(nombreURL);

        save = (FloatingActionButton) findViewById(R.id.floating);
        save.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                try {
                    save();
                } catch (Exception e) {
                    e.printStackTrace();
                }
            }
        });

    }

    private void save() {
        if (webServiceUrl.getText().toString().equals(nombreURL)) {
            Toast.makeText(this,"No se mismo " + nombreURL + "",Toast.LENGTH_SHORT).show();
            Functions.createNotification(this, 100, "Cambiando el WebService", "No se modifico el webservice ya que es el mismo");
        } else {
            if (Functions.checkURL(webServiceUrl.getText().toString())) {

                properties.setUrlWebsrv((webServiceUrl.getText().toString()));
                Toast.makeText(this,"WebService Mod " + nombreURL + "",Toast.LENGTH_SHORT).show();
                Functions.createNotification(this, 100, "Cambiando el WebService", "WebService Modificado Correctamente");
            } else {
                Toast.makeText(this,"No se mod url " + nombreURL + "",Toast.LENGTH_SHORT).show();
                Functions.createNotification(this, 100, "Cambiando el WebService", "No se modifico el webservice ya el Url Es invalido");
            }
        }
        Intent i = new Intent(getApplicationContext(), LoginActivity.class);
        startActivity(i);
    }
}

File: app/src/main/java/com/friendlypos/login/activity/ConfiguracionActivity.kt
package com.friendlysystemgroup.friendlypos.login.activity

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.TextView
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.friendlysystemgroup.friendlypos.R
import com.friendlysystemgroup.friendlypos.application.util.Functions.checkURL
import com.friendlysystemgroup.friendlypos.application.util.Functions.createNotification
import com.friendlysystemgroup.friendlypos.login.util.Properties
import com.google.android.material.floatingactionbutton.FloatingActionButton

class ConfiguracionActivity : AppCompatActivity() {
    var webServiceUrl: TextView? = null
    var save: FloatingActionButton? = null
    var properties: Properties? = null
    var nombreURL: String? = null
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_configuracion)
        properties = Properties(this)
        webServiceUrl = findViewById<View>(R.id.txtwebservice) as TextView

        nombreURL = properties!!.urlWebsrv

        webServiceUrl!!.text = nombreURL

        save = findViewById<View>(R.id.floating) as FloatingActionButton
        save!!.setOnClickListener {
            try {
                save()
            } catch (e: Exception) {
                e.printStackTrace()
            }
        }
    }

    private fun save() {
        if (webServiceUrl!!.text.toString() == nombreURL) {
            Toast.makeText(this, "No se mismo $nombreURL", Toast.LENGTH_SHORT).show()
            createNotification(
                this,
                100,
                "Cambiando el WebService",
                "No se modifico el webservice ya que es el mismo"
            )
        } else {
            if (checkURL(webServiceUrl!!.text.toString())) {
                properties!!.urlWebsrv = (webServiceUrl!!.text.toString())
                Toast.makeText(this, "WebService Mod $nombreURL", Toast.LENGTH_SHORT).show()
                createNotification(
                    this,
                    100,
                    "Cambiando el WebService",
                    "WebService Modificado Correctamente"
                )
            } else {
                Toast.makeText(this, "No se mod url $nombreURL", Toast.LENGTH_SHORT).show()
                createNotification(
                    this,
                    100,
                    "Cambiando el WebService",
                    "No se modifico el webservice ya el Url Es invalido"
                )
            }
        }
        val i = Intent(applicationContext, LoginActivity::class.java)
        startActivity(i)
    }
}